Auth0/Auth0Authentication.swift

```diff
@@ -19,11 +19,11 @@ struct Auth0Authentication: Authentication {
         self.telemetry = telemetry
     }
 
-    func login(email username: String, code otp: String, audience: String?, scope: String?) -> Request<Credentials, AuthenticationError> {
```

Comment:
With a default value, there's no point in having the `scope` parameter be optional.